sse-gateway/package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/jenkinsci/sse-gateway-plugin",
   "dependencies": {
-    "@jenkins-cd/sse-gateway": "0.0.21"
+    "@jenkins-cd/sse-gateway": "../sse-gateway-client"
```

Comment:
It probably doesn't matter too much in this case, but I guess this really is a `devDependencies` since it's only there to service unit tests.

Reply:
Updated.